core/src/main/resources/defaults/blueprints/7.2.15/cdp-rt-data-mart.bp

```diff
@@ -192,11 +192,9 @@
         "refName": "coordinator",
         "cardinality": 1,
         "roleConfigGroupsRefNames": [
-          "hdfs-DATANODE-BASE",
           "spark_on_yarn-GATEWAY-BASE",
```

Comment:
I'm still not clear on whether we really need spark or yarn services on this cluster type at all, and if we do I still question whether we should have a "spark_on_yarn-GATEWAY-BASE" and "yarn-NODEMANAGER-BASE" role on the coordinator instance.  The nodemanager in particular will use resources and compete with the coordinator.  In any case if we do need these roles on the coordinator, we may need to revisit whether we can configure the coordinator instance with only ephemeral storage and still safely stop it.  Even though these roles may be whitelisted for stopping it's possible that they're require some persistent storage to safely stop.  I think we need more input from PM (and also possibly from YARN/Spark teams) on this.

Reply:
Per the clarification from Manish in the JIRA here, we don't want the "yarn-NODEMANAGER-BASE" role on any of the coordinators so please remove it from the coordinator on all templates.  Coordinators should just have the following roles:
"spark_on_yarn-GATEWAY-BASE",
"impala-IMPALAD-COORDINATOR"